### PR TITLE
Add changes about Aadhar

### PR DESCRIPTION
**Pull request by ghost**
details and some personal intersts.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# About Me
+My name is Aadhar Toshniwal. I am an CSE student at Panjab University, India.
```
Comment (divyam3897) [inline about.md]: an CSE student?

File: about.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Experience
+Just started programming.Experiences coming soon...
```
Comment (divyam3897) [inline about.md]: space after period. remove the extra dots.

File: about.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Contact Info
+Email: [aadhar.toshniwal12@gmail.com](mailto:aadhar.toshniwal12@gmail.com@gmail.com)
+
+Facebook profile: [Aadhar Toshniwal](https://www.facebook.com/aadhar.toshniwal.5)
+
+Github: [aadhar12](https://github.com/aadhar12)
+
+# Emoji
+My favorite emoji is :smile:
+
```
Comment (divyam3897) [inline about.md]: extra line?

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Write commit messages as mentioned.

**Comment by ghost** [comment]
sir now is it fine or any other changes are required

File: about.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# About Me
+My name is Aadhar Toshniwal. I am a CSE student at Panjab University,India.
```
Comment (divyam3897) [inline about.md]: space after ","

File: about.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+Just started programming.Experiences coming soon.
+# Interests
+Open source, Machine learning and Artificial Intelligence.
+# Contact Info
+Email: [aadhar.toshniwal12@gmail.com](mailto:aadhar.toshniwal12@gmail.com@gmail.com)
+Facebook profile: [Aadhar Toshniwal](https://www.facebook.com/aadhar.toshniwal.5)
+Github: [aadhar12](https://github.com/aadhar12)
+# Emoji
+My favorite emoji is :smile:
+
```
Comment (divyam3897) [inline about.md]: This empty line?

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Squash your commits into one, then it's good with the changes suggested.

**Comment by ghost** [comment]
sir i have just squashed the commits. please could u check them once if everything is correct or not

**Comment by ghost** [comment]
sir just squashed all the commits. could u please check them if ther are
correct or not

**Review by divyam3897 — CHANGES_REQUESTED** [review]
I still don't see the last line being removed, squashing means you should just have one commit in your PR, you have 3

**Comment by ghost** [comment]
So should I have to squash all of those commits as well which were made by
u.

**Comment by divyam3897** [comment]
No you just need to squash your commits into one commit.

**Pull request closed without merging**